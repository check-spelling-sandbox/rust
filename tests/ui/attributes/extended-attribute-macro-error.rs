// normalize-stderr-test: "couldn't read.*" -> "couldn't read the file"

#![doc = include_str!("../nonexistent_file.md")]
struct Documented {}
//~^^ ERROR couldn't read

fn main() {}
