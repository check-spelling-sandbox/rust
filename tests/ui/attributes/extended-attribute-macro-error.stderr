error: couldn't read the file
  --> $DIR/extended-attribute-macro-error.rs:3:10
   |
LL | #![doc = include_str!("../nonexistent_file.md")]
   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `include_str` (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

